contrib/extractor.bat

```diff
@@ -1,57 +1,76 @@
 @ECHO OFF
-CLS
-
-:MENU
-ECHO.
-ECHO ...............................................
-ECHO Trinitycore dbc/db2, maps, vmaps, mmaps extractor
-ECHO ...............................................
-ECHO PRESS 1, 2, 3 OR 4 to select your task, or 5 to EXIT.
-ECHO ...............................................
-ECHO.
-ECHO The vmaps extractor output text below is intended and not an error:
-ECHO ..........................................
-ECHO Extracting World\Wmo\Band\Final_Stage.wmo
-ECHO No such file.
-ECHO Couldn't open RootWmo!!!
-ECHO Done!
-ECHO ..........................................
-ECHO.
-ECHO 1 - Extract dbc/db2 and maps
-ECHO 2 - Extract vmaps (needs maps to be extracted before you run this)
-ECHO 3 - Extract mmaps (needs vmaps to be extracted before you run this, may take hours)
-ECHO 4 - Extract all (may take hours)
-ECHO 5 - EXIT
-ECHO.
-SET /P M=Type 1, 2, 3, 4 or 5 then press ENTER:
-IF %M%==1 GOTO MAPS
-IF %M%==2 GOTO VMAPS
-IF %M%==3 GOTO MMAPS
-IF %M%==4 GOTO ALL
-IF %M%==5 GOTO :EOF
+title Simple Extractor
+COLOR 0A
 
-:MAPS
-start /b /w mapextractor.exe
-GOTO MENU
+CLS
+ECHO.
+ECHO     ______                       __
+ECHO    /\__  _\       __          __/\ \__
+ECHO    \/_/\ \/ _ __ /\_\    ___ /\_\ \, _\  __  __
+ECHO         \ \ \/\`'__\/\ \ /' _ `\/\ \ \ \/ /\ \/\ \
+ECHO          \ \ \ \ \/ \ \ \/\ \/\ \ \ \ \ \_\ \ \_\ \
+ECHO           \ \_\ \_\  \ \_\ \_\ \_\ \_\ \__\//`____ \
+ECHO            \/_/\/_/   \/_/\/_/\/_/\/_/\/__/ `/___/  \
+ECHO                                       C O R E  /\___/
+ECHO      http://TrinityCore.org                    \/__/
+ECHO.
+ECHO.
+ECHO.
+ECHO.
+PAUSE
 
-:VMAPS
-start /b /w vmap4extractor.exe
-start /b /w vmap4assembler.exe Buildings vmaps
-rmdir Buildings /s /q
-GOTO MENU
+CLS
+COLOR 0A
+ECHO.
+ECHO.
+ECHO                             ***NOTE*** 
+ECHO This entire process will take anywhere from 2 hours to 8 hours or MORE, 
+ECHO                depending upon the speed of your machine. (If you have a potato machine *Shauren says that*)
```

Comment:
```suggestion
ECHO depending upon the speed of your machine.
ECHO  (if you have a potato machine. *Shauren says that*)
```
You want to keep info text lines at max. 80 char. length, if end user has default size cmd window.

Reply:
i use default size and all fine, no change needed